US/CA/san-francisco.md

```diff
@@ -0,0 +1,13 @@
+## San Francisco, CA
+  - Places to eat:
```

Comment:
@githubteacher: Gary Danko's is in North Beach/Little Italy. It's a 1 Michelin star restaurant. So reserve in advance. Foreign Cinema is in the Mission District. They play an old movie in the background while you eat. Brenda's French Soul Food is in the Tenderloin.